packages/serverpod/lib/src/server/serverpod.dart

```diff
@@ -212,13 +212,26 @@ class Serverpod {
   /// started.
   late final ClusterManager cluster;
 
+  /// Define CORS headers to allow cross-origin requests.
+  final Map<String, dynamic> headers;
+
   /// Creates a new Serverpod.
   Serverpod(
     List<String> args,
     this.serializationManager,
     this.endpoints, {
     this.authenticationHandler,
     this.healthCheckHandler,
+    this.headers = const {
+      'Access-Control-Allow-Origin':
+          '*', // Allow all origins or specify a specific origin.
+      'Access-Control-Allow-Methods':
```

Comment:
We are only using `POST` in Serverpod, so maybe this should be the only option allowed?